src/gui/sdlmain.cpp

```diff
@@ -3403,6 +3403,8 @@ static void set_output(Section* sec, const bool wants_aspect_ratio_correction)
 	const auto transparency = clamp(section->Get_int("transparency"), 0, 90);
 	const auto alpha = static_cast<float>(100 - transparency) / 100.0f;
 	SDL_SetWindowOpacity(sdl.window, alpha);
+	SDL_SetWindowTitle(sdl.window, "DOSBox Staging");
+	SetIcon();
```

Comment:
Suggest using `DOSBOX_NAME`, which is a #define that comes in from the config system:

```
#define DOSBOX_NAME "DOSBox Staging"
```

(Used in a couple places)

Reply:
Pushed a new commit to address this.  There was another one called `APP_NAME_STR` that appends `(debug build)` for the debug build.

Reply:
Oh, another de-dupe - nice @weirddan455!